api/common.go

```diff
@@ -2,6 +2,13 @@ package api // import "github.com/docker/docker/api"
 
 // Common constants for daemon and client.
 const (
+	// FallbackVersion is the version to fallback to if API-version negotiation
+	// fails. This version is the highest version of the API before API-version
+	// negotiation was introduced. If negotiation fails (or no API version was
+	// included in the API response), we assume the API server uses the most
+	// recent version before negotiation was introduced.
+	FallbackVersion = "1.24"
+
 	// DefaultVersion of Current REST API
 	DefaultVersion = "1.44"
```

Comment:
Perhaps change to `Version` (if we remove "unversioned === default === current" version

Reply:
^^ didn't change this one yet, but can do so in a follow-up.